packages/web/pages/player/[username].tsx

```diff
@@ -100,6 +101,15 @@ const PlayerPage: React.FC<Props> = ({ player }) => {
 
   return (
     <PageContainer>
+      <HeadComponent
+        title={`Metagame profile for ${player.username}`}
+        description={`${player.box_profile?.description} || ''`.replace(
```

Comment:
```suggestion
        description={(player.box_profile?.description || '').replace(
```

Reply:
There are existing methods you can instead use here 
`getPlayerDescription` from `https://github.com/MetaFam/TheGame/blob/develop/packages/web/utils/playerHelpers.ts`

Event in the title instead of username you could use `getPlayerName`

Reply:
for some reason, when I add this suggestion, the automated tests in the pre-commit hook fail with -
```
@metafam/web: /Users/vyom/TheGame/packages/web/pages/player/[username].tsx
@metafam/web:   106:53  error  Parsing error: ')' expected
@metafam/web: ✖ 1 problem (1 error, 0 warnings)
```

Reply:
Though, I think using the helpers would be much cleaner :+1:

Reply:
d9a4e2a